packages/web/src/components/track/mobile/ConnectedTrackListItem.tsx

```diff
@@ -101,3 +138,13 @@ export default connect(
   mapStateToProps,
   mapDispatchToProps
 )(memo(ConnectedTrackListItem))
+function openPremiumContentPurchaseModal(
```

Comment:
holy...

Reply:
Lol sorry. Didn't get to self review this one yet. Thanks for catching it